glauth/vars/main.yaml

```diff
@@ -0,0 +1,41 @@
+# vim:ts=2:sw=2:et:ai:sts=2
+---
+# TODO: validation.
+_glauth_toplevel:
+  debug: '{{ glauth__enable_debug | bool }}'
+  syslog: '{{ glauth__log_to_syslog | bool }}'
+  structuredlog: '{{ glauth__structured_log | bool }}'
+
+_glauth_services:
+  api: '{{ glauth__service_api }}'
+  ldap: '{{ glauth__service_ldap }}'
+  ldaps: '{{ glauth__service_ldaps }}'
+
+_glauth_configs:
+  tracing: '{{ glauth__config_tracing }}'
+  backends: '{{ glauth__config_backends }}'
+  behaviors: '{{ glauth__config_behaviors }}'
+
+_glauth_datastore:
+  groups: '{{ glauth__datastore_groups }}'
+  # users filter:
+  #   - drop passsha256/passappsha256
+  #   - optionally hexlify passbcrypt
+  users: |
+      {% filter ansible.builtin.from_yaml %}
+      {% if glauth__datastore_users %}
+      {% set users = glauth__datastore_users | ansible.utils.remove_keys(
+             matching_parameter='regex', target='(?i)pass(app)?sha256$') %}
+      {% for user in users %}
+      {% set pass = user.get('passbcrypt') or '' %}
```

Comment:
Sadly, the golang toml parser allows you to ignore case (and glauth does), but I think we should only use the canonical camelCase names

Reply:
How about only using lowercase, which is never ambiguous? (_passBcrypt?_ _passBCrypt?_). As far as I could see, config examples in the glauth repo almost exclusively use lower case for `[[users]]`.

Reply:
Uhm, it is a good point, but it is not very *elegant*, and still allows for confusing bugs if you don' t actually use lowercase everywhere.. Buuut, one good solution would be to force all keys to lowercase before proceeding, like this:

```
{{ foo.keys() | map('lower') | zip(foo.values()) | community.general.dict }}
```

What do you think?

Reply:
Sure, so I'd do that for users and groups only, right?

Reply:
I'd do it for all vars that include config keys

Reply:
Okay. I'll make use of your LGTM now, and do this lowercasing in a follow-up PR (together with tests). I want to get the base role out of my head! (And usable already by others.)